MOBILE_API.xml

```diff
@@ -3699,7 +3688,428 @@
+
+        <param name="mediaServiceData" type="MediaServiceData" mandatory="false"/>
+        <param name="weatherServiceData" type="WeatherServiceData" mandatory="false"/>
+        <param name="navigationServiceData" type="NavigationServiceData" mandatory="false"/>
+    </struct>
+
+    <enum name="ServiceUpdateReason" since="5.1">
+        <element name="PUBLISHED">
+            <description> The service has just been published with the module and once activated to the primary service of its type, it will be ready for possible consumption.</description>
+        </element>
+        <element name="REMOVED">
+            <description> The service has just been unpublished with the module and is no longer accessible</description>
+        </element>
+        <element name="ACTIVATED">
+            <description> The service is activated as the primary service of this type. All requests dealing with this service type will be handled by this service.</description>
+        </element>
+        <element name="DEACTIVATED">
+            <description> The service has been deactivated as the primary service of its type</description>
+        </element>
+        <element name="MANIFEST_UPDATE">
+            <description> The service has updated its manifest. This could imply updated capabilities</description>
+        </element>
+    </enum>
+
+    <struct name="AppServiceCapability" since="5.1">
+        <param name="updateReason" type="ServiceUpdateReason" mandatory="false">
+            <description> Only included in OnSystemCapabilityUpdated. Update reason for service record.</description>
+        </param>
+        <param name="updatedAppServiceRecord" type="AppServiceRecord" mandatory="true">
+            <description>Service record for a specific app service provider</description>
+        </param>
+    </struct>
+
+    <struct name="AppServicesCapabilities" since="5.1">
+        <description>Capabilities of app services including what service types are supported and the current state of services.</description>
+        <param name="appServices" type="AppServiceCapability" array="true" mandatory="false">
+            <description>An array of currently available services. If this is an update to the capability the affected services will include an update reason in that item</description>
+        </param>
+    </struct>
+
+    <!-- End App Services -->
     
+    <struct name="SystemCapability" since="4.5">
+        <description>The systemCapabilityType identifies which data object exists in this struct. For example, if the SystemCapability Type is NAVIGATION then a "navigationCapability" should exist</description>
+        <param name="systemCapabilityType" type="SystemCapabilityType" mandatory="true">
+            <description>Used as a descriptor of what data to expect in this struct. The corresponding param to this enum should be included and the only other para included.</description>
```

Comment:
_ para_ should be _ param_

Reply:
Fixed in https://github.com/smartdevicelink/rpc_spec/pull/149/commits/06a3394c636aa24d9b880a352c67c94700476e0e